packages/os/disable-udp-offload.service

```diff
@@ -0,0 +1,22 @@
+# This unit disables UDP offload for the default primary interface (eth0) in
+# VMware.  This avoids an issue where newer kernels (5.10+) and the vmxnet3
+# driver conflict with NSX-T.
+[Unit]
+Description=Disables UDP offload
+After=network-online.target
+# Run before bootstrap containers so a user could reverse the change via
+# bootstrap container if they desired.
+Before=bootstrap-containers@.service
+# Block manual interactions with this service
+RefuseManualStart=true
+RefuseManualStop=true
+
+[Service]
+Type=oneshot
+ExecStart=/usr/sbin/ethtool -K eth0 tx-udp_tnl-segmentation off
+ExecStart=/usr/sbin/ethtool -K eth0 tx-udp_tnl-csum-segmentation off
```

Comment:
Can you verify that `ethtool -k eth0` on an existing image has "on" values for both these settings?

Reply:
Yep - I booted a VMware image without the systemd unit and verified the settings show as "on".  Updated the PR description as well:
```
bash-5.1# ethtool -k eth0 | grep -i tx-udp
tx-udp_tnl-segmentation: on
tx-udp_tnl-csum-segmentation: on
```